test/test_rosbag_image_compressor.py

```diff
@@ -107,8 +108,23 @@ def test_bag_unbag(self):
         with TestBag(4) as bag_context:
             intermediate = os.path.join(bag_context.tempdir, 'intermediate.bag')
             output = os.path.join(bag_context.tempdir, 'output.bag')
+
+            # run compress on the bag
             compress(bag_context.filename, intermediate)
+
+            # check intermediate bag
+            with rosbag.Bag(intermediate, 'r') as bag:
+                for topic, msg, t in bag:
+                    # non-standard topic name should have flag in encoding
+                    if topic == '/foo':
```

Comment:
I think you're correct. It works with either topic `foo` and then checking `msg.encoding.endswith()` or using topic `foo/encoding` and then checking `msg.endswith()`... I'll change this